apps/studio/src/features/editing-experience/components/LayoutSelection.tsx

```diff
@@ -0,0 +1,281 @@
+              {`Choose a layout for "`}
+              {props.pageName}
+              {`"`}
+            </Text>
+            <Button size="xs">
+              Start with this layout
+              <BiRightArrowAlt fontSize="1.5rem" />
+            </Button>
+          </HStack>
+        </VStack>
+        <Grid
+          w="100%"
+          h="calc(100% - 7rem)"
+          templateColumns="repeat(4, 1fr)"
+          gap={0}
+        >
+          <GridItem w="100%" colSpan={1} bg="grey.50" overflowY="scroll">
+            <VStack p="2rem" gap="2rem" h="fit-content">
+              {/* Picking Layouts */}
+              {LAYOUT_DATA.map((layoutEntry) => {
+                return (
+                  <VStack
+                    key={layoutEntry.layoutTypename}
+                    p="0"
+                    gap="0"
+                    cursor="pointer"
+                    onClick={() => {
+                      setSelectedLayout(layoutEntry)
+                    }}
+                    role="group"
+                  >
+                    <Box
+                      className="hover-border"
+                      borderRadius="8px"
+                      border="2px"
+                      p="1.5rem 1.5rem 0 1.5rem"
+                      mb="1.25rem"
+                      position="relative"
+                      borderColor={
+                        selectedLayout.layoutTypename ===
+                        layoutEntry.layoutTypename
+                          ? 'base.divider.brand'
+                          : 'base.divider.medium'
+                      }
+                      bgColor={
+                        selectedLayout.layoutTypename ===
+                        layoutEntry.layoutTypename
+                          ? 'interaction.muted.main.active'
+                          : 'interaction.muted.main.hover'
+                      }
+                      sx={
+                        selectedLayout.layoutTypename !==
+                        layoutEntry.layoutTypename
+                          ? {
+                              transition:
+                                'border-color 300ms ease-out, opacity 300ms ease-out, background-color 300ms ease-out',
+                            }
+                          : {}
+                      }
+                      _groupHover={
+                        selectedLayout.layoutTypename !==
+                        layoutEntry.layoutTypename
+                          ? {
+                              borderColor: useToken(
+                                'colors',
+                                'interaction.main-subtle.hover',
+                              ),
+                              bgColor: useToken(
+                                'colors',
+                                'interaction.main-subtle.default',
+                              ),
+                            }
+                          : {}
+                      }
+                    >
```

Comment:
comment(chakra-specific): Might want to style active state directly instead of doing a bunch of conditionals

see: [chakra docs](https://v2.chakra-ui.com/docs/styled-system/style-props#:~:text=none-,_active,-%26%3Aactive%0A%26%5Bdata%2Dactive)

```suggestion
                    <Box
                      className="hover-border"
                      borderRadius="8px"
                      border="2px"
                      p="1.5rem 1.5rem 0 1.5rem"
                      mb="1.25rem"
                      position="relative"
                      data-active={
                        selectedLayout.layoutTypename ===
                        layoutEntry.layoutTypename
                      }
                      _active={{
                        borderColor: 'base.divider.brand',
                        bgColor: 'interaction.muted.main.active',
                        transition:
                          'border-color 300ms ease-out, opacity 300ms ease-out, background-color 300ms ease-out',
                        _groupHover: {
                          bgColor: 'interaction.main-subtle.default',
                        },
                      }}
                      borderColor="base.divider.medium"
                      bgColor="interaction.muted.main.hover"
                      _groupHover={{
                        borderColor: 'interaction.main-subtle.hover',
                      }}
                    >
```